<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>627</width>
            <height>653</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="3" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Negative prompt</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Propmpt</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="run_riffusion">
                <property name="text">
                 <string>Run Riffusion</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="stop_riffusion">
                <property name="text">
                 <string>Stop Riffusion</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QPlainTextEdit" name="prompt"/>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QPlainTextEdit" name="neg_prompt"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Seed image</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="use_alpha_as_mask_2">
                <property name="text">
                 <string>Use alpha as mask</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="invert_mask_2">
                <property name="text">
                 <string>Invert mask</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>Use mask</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Mask image</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QCheckBox" name="overlay_mask_2">
                <property name="text">
                 <string>Overlay Mask</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="QComboBox" name="selected_mask_image"/>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QComboBox" name="selected_seed_image"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="sampler_values">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>5</number>
              </property>
              <item row="21" column="0" colspan="2">
               <widget class="QCheckBox" name="show_output_setup">
                <property name="text">
                 <string>Output Options</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_72">
                <property name="text">
                 <string>Model</string>
                </property>
               </widget>
              </item>
              <item row="16" column="2">
               <widget class="QDoubleSpinBox" name="ddim_eta">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="12" column="3">
               <widget class="QCheckBox" name="show_sample_per_step">
                <property name="text">
                 <string>Live</string>
                </property>
               </widget>
              </item>
              <item row="7" column="2" colspan="4">
               <widget class="QComboBox" name="selected_model"/>
              </item>
              <item row="12" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Batch</string>
                </property>
               </widget>
              </item>
              <item row="14" column="3" colspan="2">
               <widget class="QCheckBox" name="hires">
                <property name="text">
                 <string>HiRes Mode</string>
                </property>
               </widget>
              </item>
              <item row="11" column="2" colspan="3">
               <widget class="QLineEdit" name="seed"/>
              </item>
              <item row="14" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Strength</string>
                </property>
               </widget>
              </item>
              <item row="12" column="2">
               <widget class="QSpinBox" name="n_batch">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>2000</number>
                </property>
               </widget>
              </item>
              <item row="16" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>DDIM ETA</string>
                </property>
               </widget>
              </item>
              <item row="11" column="6">
               <widget class="QComboBox" name="seed_behavior">
                <item>
                 <property name="text">
                  <string>iter</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>random</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>fixed</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="14" column="5">
               <widget class="QCheckBox" name="seamless">
                <property name="text">
                 <string>Seamless</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Sampler:</string>
                </property>
               </widget>
              </item>
              <item row="6" column="6">
               <widget class="QPushButton" name="update_models">
                <property name="minimumSize">
                 <size>
                  <width>110</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Update Models</string>
                </property>
               </widget>
              </item>
              <item row="11" column="5">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Mode</string>
                </property>
               </widget>
              </item>
              <item row="12" column="5">
               <widget class="QCheckBox" name="karras">
                <property name="text">
                 <string>Karras</string>
                </property>
               </widget>
              </item>
              <item row="7" column="6">
               <widget class="QPushButton" name="load_model">
                <property name="text">
                 <string>Load Model</string>
                </property>
               </widget>
              </item>
              <item row="22" column="0" colspan="7">
               <widget class="QFrame" name="output_values">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <property name="leftMargin">
                  <number>3</number>
                 </property>
                 <property name="topMargin">
                  <number>3</number>
                 </property>
                 <property name="rightMargin">
                  <number>3</number>
                 </property>
                 <property name="bottomMargin">
                  <number>3</number>
                 </property>
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item row="5" column="1" colspan="2">
                  <widget class="QLineEdit" name="outdir">
                   <property name="text">
                    <string>data/output/riffusion</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="label_67">
                   <property name="text">
                    <string>Output Path</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QCheckBox" name="save_settings">
                   <property name="text">
                    <string>Save Settings</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="4">
                  <widget class="QPushButton" name="select_outdir">
                   <property name="text">
                    <string>Select Path</string>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>8</width>
                     <height>8</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QCheckBox" name="save_samples">
                   <property name="text">
                    <string>Save Samples</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="14" column="2">
               <widget class="QDoubleSpinBox" name="strength">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item row="6" column="2" colspan="4">
               <widget class="QComboBox" name="sampler">
                <item>
                 <property name="text">
                  <string>Euler</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Euler Ancestral</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DPM</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>LMS</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DPM 2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DPM 2 Ancestral</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Heun</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DPM Fast</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DPMPP SDE</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DPMPP 2S Ancestral</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DPMPP 2M</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ddim</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>plms</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="16" column="6">
               <widget class="QComboBox" name="axis">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Both</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="16" column="5">
               <widget class="QLabel" name="label_74">
                <property name="text">
                 <string>Axis</string>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Seed</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
